daemons/dss-events-scribe/.chalice/config.json

```diff
@@ -7,57 +7,11 @@
-        "env": "dev"
-      },
-      "environment_variables": {
-        "ADMIN_USER_EMAILS": "org-humancellatlas-dev@human-cell-atlas-travis-test.iam.gserviceaccount.com,lblauvel-service-account@human-cell-atlas-travis-test.iam.gserviceaccount.com,travis-test@human-cell-atlas-travis-test.iam.gserviceaccount.com,lblauvel@ucsc.edu,bhannafi@ucsc.edu,amarjandu@human-cell-atlas-travis-test.iam.gserviceaccount.com,travis-test@human-cell-atlas-travis-test.iam.gserviceaccount.com",
-        "API_DOMAIN_NAME": "dss.dev.data.humancellatlas.org",
-        "AUTH_URL": "https://auth.testing.data.humancellatlas.org",
-        "CHECKOUT_CACHE_CRITERIA": "[{\"type\":\"application/json\",\"max_size\":12314}]",
-        "DSS_AUTHORIZED_DOMAINS": "human-cell-atlas-travis-test.iam.gserviceaccount.com broad-dsde-mint-dev.iam.gserviceaccount.com broad-dsde-mint-test.iam.gserviceaccount.com broad-dsde-mint-staging.iam.gserviceaccount.com",
-        "DSS_AWS_FLASHFLOOD_PREFIX": "aws",
-        "DSS_AWS_FLASHFLOOD_PREFIX_READ": "aws",
-        "DSS_AWS_FLASHFLOOD_PREFIX_WRITE": "aws",
-        "DSS_BLOB_PUBLIC_TTL_DAYS": "7",
-        "DSS_BLOB_TTL_DAYS": "14",
-        "DSS_BUNDLE_MANIFEST_INDEX_LIMIT": "20000",
-        "DSS_DEBUG": "0",
-        "DSS_DEPLOYMENT_STAGE": "dev",
-        "DSS_ES_ENDPOINT": "search-dss-index-dev-wc7kz2vuj7mpfjbxnp4ftplpuu.us-east-1.es.amazonaws.com",
-        "DSS_EVENT_RELAY_NAME": "dss-gs-event-relay-",
-        "DSS_FLASHFLOOD_BUCKET": "org-humancellatlas-dss-events-dev",
-        "DSS_GCP_FLASHFLOOD_PREFIX": "gcp",
-        "DSS_GCP_FLASHFLOOD_PREFIX_READ": "gcp",
-        "DSS_GCP_FLASHFLOOD_PREFIX_WRITE": "gcp",
-        "DSS_GS_BUCKET": "org-humancellatlas-dss-dev",
-        "DSS_GS_CHECKOUT_BUCKET": "org-humancellatlas-dss-checkout-dev",
-        "DSS_NOTIFICATION_SENDER": "DSS Notifier <dss.humancellatlas@gmail.com>",
-        "DSS_NOTIFY_DELAYS": "0 60 600 3600 21600 60540 86400 86400 86400 86400 86400 86400",
-        "DSS_NOTIFY_TIMEOUT": "60",
-        "DSS_S3_BUCKET": "org-humancellatlas-dss-dev",
-        "DSS_S3_CHECKOUT_BUCKET": "org-humancellatlas-dss-checkout-dev",
-        "DSS_VERSION": "061e71e5027bc94f61f3adea8834a076d4bce1b2",
-        "DSS_XRAY_TRACE": "0",
-        "GCP_DEFAULT_REGION": "us-central1",
-        "GCP_PROJECT_NAME": "human-cell-atlas-travis-test",
-        "OIDC_AUDIENCE": "https://dev.data.humancellatlas.org/,https://dev.data.humancellatlas.org/",
-        "OIDC_EMAIL_CLAIM": "https://auth.data.humancellatlas.org/email",
-        "OIDC_GROUP_CLAIM": "https://auth.data.humancellatlas.org/group",
-        "OPENID_PROVIDER": "https://humancellatlas.auth0.com/",
-        "TOKENINFO_FUNC": "dss.util.security.verify_jwt"
-      },
-      "layers": [
-        "arn:aws:lambda:us-east-1:861229788715:layer:dss-dependencies-dev:45"
-      ]
+      "tags": {},
+      "environment_variables": {}
     }
   },
   "reserved_concurrency": 1,
-  "lambda_timeout": 300,
+  "lambda_timeout": 600,
```

Comment:
if the lambda is processing events, and the lambda gets a timeout due to long duration, what happens to the events that it was working on? Is there a potential for malformed journals?

Reply:
Yes, interrupting the journaling process will produce malformed journals. Currently, malformed journals will need to be created from the main bucket.

It is possible to build recovery capabilities into flashflood, which should be done in the future.